pkg/sql/logictest/testdata/logic_test/enums

```diff
@@ -359,3 +359,16 @@ query T
 SELECT x FROM greeting_stats
 ----
 hi
+
+# Test that we can cast from enum to bytes.
+# Use a singleton enum so that the bytes encoding is simple.
```

Comment:
Isn't this testing that you can cast from bytes to enum, not the opposite? We should have a test of the opposite direction too, unless I'm missing something.

Reply:
Woops, I had this backwards. I'm not sure if I want to expose casting from enum to bytes. I want to use it just internally, but I guess there isn't really a harm?

Reply:
does psql support bytes to enum or vice versa?

Reply:
no, this is just for our internal representation, since we use bytes.